webpack/index.js

```diff
@@ -76,13 +94,49 @@ const common = ({ platform }) => ({
     },
 });
 
-module.exports = Promise.all(
-    ['browser', 'server'].map(async platform =>
-        merge(
-            await require(`./${platform}`)(),
-            common({
-                platform,
-            }),
-        ),
-    ),
+module.exports = getSites().then(sites =>
+    sites.reduce(async (configs, site) => {
+        const pages = await getPagesForSite(site);
+        return [
+            // server bundle config
+            merge(
+                {
+                    entry: {
+                        [`${site}.server`]: require.resolve(
+                            `@guardian/rendering/server`,
```

Comment:
Does this need to be a template string?

Reply:
Why not `require()` at the top of the module? Looks like it's just exporting a singleton

Reply:
this just returns the path, which webpack then uses to create the bundle. i'm not sure webpack will work with the export here?

Reply:
Right on, I am drunk